apps/microservices-orders/orders-service/Makefile

```diff
@@ -0,0 +1,28 @@
+.PHONY: *
+
+HELP_TAB_WIDTH = 25
+
+.DEFAULT_GOAL := help
+
+SHELL=/bin/bash -o pipefail
+
+check-dependency = $(if $(shell command -v $(1)),,$(error Make sure $(1) is installed))
+
+clean:
+	@./gradlew clean
+
+test:
+	@./gradlew test
+
+build:
+	@./gradlew shadowJar
+
+package:
+	@./gradlew jibDockerBuild
```

Comment:
springboot plugin has the built-in capability to building image - `./gradlew bootBuildImage`